src/Symfony/Component/Messenger/Bridge/Redis/Transport/RedisProxy.php

```diff
@@ -33,7 +33,10 @@ public function __construct(\Redis $redis, \Closure $initializer)
 
     public function __call(string $method, array $args)
     {
-        $this->ready ?: $this->ready = $this->initializer->__invoke($this->redis);
+        if (!$this->ready) {
+            $this->redis = $this->initializer->__invoke($this->redis);
+            $this->ready = true;
+        }
```

Comment:
Maybe Im missing something, but can you tell me why this was changed?

Reply:
that's because the signature of the initializer has changed (this is fine because this is internal)